### ISSUE TITLE
Error: Could not find or load main class org.aksw.simba.bengal.controller.BengalController

### ISSUE THREAD

**Issue by vigneshBlockDev** [dice-group/BENGAL#17]
I did the mvn clean &&  copy paste all the extracted file in the desired location..

then i tried executed this line 

**java -cp ./target/BENGAL-1.0-SNAPSHOT-exec.jar org.aksw.simba.bengal.controller.BengalController -st sym -dp**

Am **using the windows** for this...!!

but still getting the **Could not find or load main class**..

please guide me...!!!

**Comment by MichaelRoeder** [comment]
Please make sure that:
1. ... you run `mvn clean install` (not only clean :wink: )
2. ... the command terminates successfully (i.e., you should see something like `[INFO] BUILD SUCCESS` at the end)
3. ... the `target` directory contains the file `BENGAL-1.0-SNAPSHOT-exec.jar`

**Comment by vigneshBlockDev** [comment]
@MichaelRoeder 
Sry for the typo i did the mv clean install only..
I got info like **BUILD SUCCESS**.
after that in the target folder i am not seeing any **BENGAL-1.0-SNAPSHOT-exec.jar** in this name.

i have three file with following name
![Screenshot (39)](https://user-images.githubusercontent.com/40856997/86235939-dcbc6e80-bbb6-11ea-974a-a4e93babbab5.png)

- BENGAL-1.0-SNAPSHOT.jar

- BENGAL-1.0-SNAPSHOT-javadoc.jar

- BENGAL-1.0-SNAPSHOT-sources.jar


Is there any changes should i make in the cmd i am running?

Kindly help me in this regard.

**Comment by MichaelRoeder** [comment]
Yes, there was an issue with the configuration and the documentation of the build command. Please pull the latest changes and execute
```
mvn clean install shade:shade
```
Please let us know whether this fixed the problem.

**Comment by vigneshBlockDev** [comment]
@MichaelRoeder 

Thanks for the back to back help..!

After the executing the java cp command

- Using folder C:\Users\vigne\AppData\Local\Temp\/triple2nl-cache/dereferenced as cache for IRI converter.

- Adding annotator tokenize

- No tokenizer type provided. Defaulting to PTBTokenizer.


- Adding annotator ssplit

- Adding annotator pos

- Loading POS tagger from edu/stanford/nlp/models/pos-tagger/english-left3words/english-left3words-distsim.tagger ... done [1.0 sec].

- Adding annotator lemma

- Adding annotator parse

- Loading parser from serialized file edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz ... done [0.5 sec].

- Created document #1



Where can i see the created document?

Before that 

FYI, **i am looking for any Open source ready made code that will do the paraphrasing by giving the sentence and getting output as paraphrased sentence**  when am doing google using the keywords i got to know your repository..  

So _will this code will do the above mentioned task_? becoz i am **not able to see the visible use of this code** in the readme.md

for past two days am trying to understand use of this repo by running but   am not getting...!!

please solve my doubt..before approaching further...!

**Comment by MichaelRoeder** [comment]
This project does exactly what is mentioned at the beginning of the README :wink: 

> Automatic Generation of Benchmarks for Entity Recognition and Linking
> 
> This project aims at delivering a framework for generating bechmarking data sets from structured data to the Named Entity Recognition (NER) and Disambiguation (NED a.k.a Entity Linking and Wikification) problems.

Internally, it can make use of algorithms to paraphrase a generated sentence. However, it has not been created for solely paraphrasing something and, hence, its command line interface does not support other tasks than the generation of artificial documents for benchmarking.

@DiegoMoussallem do you think the classes which do the paraphrasing can be reused, easily?

**Comment by DiegoMoussallem** [comment]
Hi @MichaelRoeder @vigneshBlockDev , 

In case you want to run the paraphrase module, you need to use its parameters -> ` -pp,--paraphrase`. However, this paraphrase is in the context of RDF triples, so it is specific for Natural Language Generation from RDF Knowledge Graphs. You can try to reuse some parts of the code but it is not applicable to other tasks.
